crates/router/src/db/refund.rs

```diff
@@ -770,4 +820,15 @@ impl RefundInterface for MockDb {
             .cloned()
             .collect::<Vec<_>>())
     }
+
+    #[cfg(feature = "olap")]
+    async fn filter_refund_by_meta_constraints(
+        &self,
+        _merchant_id: &str,
+        _refund_details: &api_models::refunds::RefundListRequest,
+        _storage_scheme: enums::MerchantStorageScheme,
+        _limit: i64,
+    ) -> CustomResult<api_models::refunds::RefundListMetaData, errors::StorageError> {
+        Err(errors::StorageError::MockDbError)?
```

Comment:
Please implement this, literally everything else in the `MockDb` implementation of `RefundInterface` is complete.